google/cloud/contactcenterinsights/contact_center_insights_connection.cc

```diff
@@ -0,0 +1,311 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated by the Codegen C++ plugin.
+// If you make any local changes, they will be lost.
+// source: google/cloud/contactcenterinsights/v1/contact_center_insights.proto
+
+#include "google/cloud/contactcenterinsights/contact_center_insights_connection.h"
+#include "google/cloud/contactcenterinsights/contact_center_insights_options.h"
+#include "google/cloud/contactcenterinsights/internal/contact_center_insights_connection_impl.h"
+#include "google/cloud/contactcenterinsights/internal/contact_center_insights_option_defaults.h"
+#include "google/cloud/contactcenterinsights/internal/contact_center_insights_stub_factory.h"
+#include "google/cloud/background_threads.h"
+#include "google/cloud/common_options.h"
+#include "google/cloud/grpc_options.h"
+#include "google/cloud/internal/async_long_running_operation.h"
+#include "google/cloud/internal/pagination_range.h"
+#include <memory>
+
+namespace google {
+namespace cloud {
+namespace contactcenterinsights {
+GOOGLE_CLOUD_CPP_INLINE_NAMESPACE_BEGIN
+
+ContactCenterInsightsConnection::~ContactCenterInsightsConnection() = default;
+
+StatusOr<google::cloud::contactcenterinsights::v1::Conversation>
+ContactCenterInsightsConnection::CreateConversation(
+    google::cloud::contactcenterinsights::v1::
+        CreateConversationRequest const&) {
+  return Status(StatusCode::kUnimplemented, "not implemented");
+}
+
+StatusOr<google::cloud::contactcenterinsights::v1::Conversation>
+ContactCenterInsightsConnection::UpdateConversation(
+    google::cloud::contactcenterinsights::v1::
+        UpdateConversationRequest const&) {
+  return Status(StatusCode::kUnimplemented, "not implemented");
+}
+
+StatusOr<google::cloud::contactcenterinsights::v1::Conversation>
+ContactCenterInsightsConnection::GetConversation(
+    google::cloud::contactcenterinsights::v1::GetConversationRequest const&) {
+  return Status(StatusCode::kUnimplemented, "not implemented");
+}
+
+StreamRange<google::cloud::contactcenterinsights::v1::Conversation>
+    ContactCenterInsightsConnection::ListConversations(
```

Comment:
FWIW, it looks like we've stumbled upon a `clang-format` bug.

Reply:
ew